test/unit/test.js

```diff
@@ -1,10 +0,0 @@
-var assert = require('chai').assert;
-
-describe('Array', function() {
-    describe('#indexOf()', function () {
-        it('should return -1 when the value is not present', function () {
-            assert.equal(-1, [1, 2, 3].indexOf(5));
-            assert.equal(-1, [1, 2, 3].indexOf(0));
-        });
-    });
-});
```

Comment:
Is this not a good hello world example, even if it isn't being called. Also this is refered to in the docs...